tests/com/github/bazelbuild/rules_jvm_external/resolver/gradle/BUILD.bazel

```diff
@@ -0,0 +1,58 @@
+load("@rules_java//java:defs.bzl", "java_test")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+java_test(
+    name = "GradleResolverTest",
+    timeout = "eternal",
```

Comment:
Increased the timeout here since the target takes about 5-6 minutes to complete as using an isolated instance of gradle per test and running things serially adds up quite a bit. We could look into test sharding if this is unacceptably slow.